libs/ui-toolkit/src/components/input-error/input-error.tsx

```diff
@@ -35,7 +35,7 @@ export const InputError = ({
     'fill-intent-warning': intent === 'warning',
   });
   return (
-    <div className={effectiveClassName} {...props}>
+    <div data-testid="error-text" className={effectiveClassName} {...props}>
```

Comment:
can we prefix this with the component name or similar, bit too generic IMO

Reply:
Added component name